### DISCUSSION TITLE
Acessibilidade na web

### DISCUSSION THREAD

**Discussion by Eduardoalves88** [frontendbr/forum#1143] [Mostre seu trabalho]
Olá!
Estou fazendo meu TCC. O tema é acessibilidade na web. Para sua conclusão é fundamental que eu faça uma pesquisa sobre o conhecimento dos desenvolvedores web sobre os padrões de acessibilidade para web. São apenas 8 perguntas a serem respondidas. 

Link para pesquisa:
https://docs.google.com/forms/d/1xCjsz61m2KFqBApqs2gT3xHE8g5cSGpPvhBaFRNt-dQ

Obrigado!

**Comment by lucianobarauna** [comment]
Opa respondo com muito prazer ;-)
Hoje estou trabalhando em um projeto com requisitos de acessibilidade.

**Comment by Eduardoalves88** [comment]
Obrigado!

**Comment by fabiorocha** [comment]
Respondido. Acessibilidade é importante demais, parabéns pelo tema escolhido pro TCC!

**Comment by linconkusunoki** [comment]
Respondido também ;)

**Comment by didiraja** [comment]
Procure o [Natan Souza](https://twitter.com/designernatan) ele tem trabalhado muito acerca de Acessibilidade
